iOSDevUK/Utils/LocationService.swift

```diff
@@ -8,17 +8,20 @@
 import MapKit
 
 enum MapDetails {
-    static let startingLocation = CLLocationCoordinate2D(latitude: 52.41483885670968, longitude: -4.076185527558135)
+    static let startingLatitude: CLLocationDegrees = 52.41483885670968
+    static let startingLongitude: CLLocationDegrees =  -4.076185527558135
+    static let defaultLocation = CLLocation(latitude: startingLatitude, longitude: startingLongitude)
+    static let default2DLocation = defaultLocation.coordinate
```

Comment:
call it like aberystwythCoordinates so tomorrow the code reader doesnt have to guess what is default2d :).